packages/app/src/cli/models/extensions/specifications/validation/app_config_webhook.ts

```diff
@@ -76,3 +87,10 @@ function validateSubscriptions(webhookConfig: WebhooksConfig) {
     }
   }
 }
+
```

Comment:
@shauns I would like to get your early opinion on api version checking like this? I half wanted to add a general "api changes support" where we have a toml (?) file that we define minimum versions for changes and allow checking for that, but this is really the only place that currently needs this sort of support.

Any thoughts are appreciated.